Sources/Internal/GrpcClient.swift

```diff
@@ -148,7 +148,12 @@ actor GrpcClient: CustomStringConvertible {
       let resultsString = try results.jsonString()
       DataConnectLogger
         .debug("executeQuery() receives response: \(resultsString, privacy: .private).")
+
       // Not doing error decoding here
+      guard results.errors.isEmpty else {
+        throw DataConnectError.operationExecutionFailed
```

Comment:
Is there a way to include the messages from the errors in the exception? That would make it infinitely easier to debug if this exception gets thrown in production.

Reply:
I've added logic to collect messages from all the errors in the list and include it as a json array.